src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -206,6 +202,21 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#iprangefiltering=<0 or 1>");
         }
 
+        private static int CalculateBanTime(Network network)
+        {
+            int banTime = 0;
+
+            // Ensure that the network's MaxReorgLength always has a value.
+            int maxReorgLength = network.Consensus.MaxReorgLength == 0 ? Network.FallBackMaxReorg : (int)network.Consensus.MaxReorgLength;
+
+            if (network.IsBitcoin())
```

Comment:
If we can move TargetSpacing to Network then we don't need this check? @dangershony @quantumagi ?